pkg/backuphelpers/backupvars_test.go

```diff
@@ -0,0 +1,127 @@
+package backuphelpers
+
+import (
+	"testing"
+
+	backupv1alpha1 "github.com/openshift/api/config/v1alpha1"
+	prune "github.com/openshift/cluster-etcd-operator/pkg/cmd/prune-backups"
+
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	schedule = "0 */2 * * *"
+	timezone = "GMT"
+)
+
+func TestBackupConfig_ToArgs(t *testing.T) {
+	testCases := []struct {
+		name     string
+		cr       *backupv1alpha1.EtcdBackupSpec
+		enabled  bool
+		expected string
+	}{
+		{
+			"enabled",
+			createEtcdBackupSpec(timezone, schedule),
+			true,
+			"    args:\n    - --enabled=true\n    - --timezone=GMT\n    - --schedule=0 */2 * * *",
+		},
+		{
+			"disabled",
+			createEtcdBackupSpec(timezone, schedule),
+			false,
+			"    args:\n    - --enabled=true\n    - --timezone=GMT\n    - --schedule=0 */2 * * *",
```

Comment:
this is confusing, you say enabled=false and it ends up as `--enabled=true`?

Reply:
so the initial state is `false`, but it will be enabled

Reply:
then we need to assert the actual output, not the initial state - otherwise the test makes no sense?

Reply:
I actually removed the `bool` from the input, the code now handles this flag according to the spec being `nil`